api/client/client_test.go

```diff
@@ -693,7 +713,6 @@ const (
 	"$schema": "http://json-schema.org/draft-07/schema#",
 	"title": "` + collectionName + `",
 	"type": "object",
-	"required": ["_id"],
```

Comment:
I removed this from the schema since it shouldn't be required of the user to demand that this is "required", but also since it was preventing the issue from being tested, since the schema validation would return an error before reaching the fatal error.